data/gBarLineDate.json

```diff
@@ -0,0 +1,15 @@
+[{"id":0,"type":132,"xval":"Aug 20","yval":74.928,"name":"Total Average","breakdown":"1634","breakdowntype":"YW","parentrefid":163,"parentreftype":135,"zval":1300},
```

Comment:
There are already two data files in our folder. groupedbarstring.json and groupedbaryw.json